src/app/auth/components/registration/registration.component.html

```diff
@@ -0,0 +1,66 @@
+<div>
+  <h5>Введите данные для регистрации</h5>
+  <h6>Поля обязательные для заполнения (<span class="color-red">*</span>)</h6>
+  <form [formGroup]="form" (ngSubmit)="onSubmit()">
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('email').invalid && form.get('email').touched ? 'color-red' : ''">*</span>
+        <app-input [formControlName]="inputFields.email"
+                   [name]="inputFields.email"
+                   [placeholder]="inputFields.email"
+        >
+        </app-input>
+      </div>
+    </div>
+    <div class="form-group">
+      <div class="form-input">
+        <span [ngClass]="form.get('password').invalid && form.get('password').touched ? 'color-red' : ''">*</span>
+        <app-input [type]="inputFields.password"
+                   [placeholder]="inputFields.password"
+                   [formControlName]="inputFields.password"
+                   [name]="inputFields.password"
+        >
+        </app-input>
+      </div>
+    </div>
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('name').invalid && form.get('name').touched ? 'color-red' : ''">*</span>
+        <app-input [placeholder]="inputFields.name"
+                   [formControlName]="inputFields.name"
+                   [name]="inputFields.name"
+        >
+        </app-input>
+      </div>
+    </div>
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('surname').invalid && form.get('surname').touched ? 'color-red' : ''">*</span>
+        <app-input [placeholder]="inputFields.surname"
+                   [formControlName]="inputFields.surname"
+                   [name]="inputFields.surname"
+        >
+        </app-input>
+      </div>
+    </div>
+
+    <div>
+      <div class="form-input">
+        <span [ngClass]="form.get('agree').invalid && form.get('agree').touched ? 'color-red' : ''">*</span>
+        <app-checkbox [formControlName]="inputFields.agree"
+                      [name]="inputFields.agree"
+                      [label]="'Agree'"
+        >
+        </app-checkbox>
+      </div>
+    </div>
+
+    <div class="">
+      <button type="submit" class="button" [disabled]="form.invalid">Регистрация</button>
+    </div>
+
+    <p class=" text-xs-centr">
+      Зарегистрированы? <span class="span-butt" (click)="toLogin()">Войти</span>
```

Comment:
span-button либо span-btn либо btn-***
```suggestion
      Зарегистрированы? <span class="span-button" (click)="toLogin()">Войти</span>
```